### ISSUE TITLE
increasing accessibility interactions in highcharts

### ISSUE THREAD

**Issue by venkateshpotluri** [highcharts/highcharts#7071]
#### Expected behaviour
highchart's interactive chart example currency rate charts, interactive piecharts are not accessible using a screen reader. Most sighted users use the mouse and watch the values change on the tooltip. But this information is not accessible to a screen reader user. The slider which works with the mouse should be adjustable using keyboard commands allowing screen readers to interact with the graph. Also, the result should be an ARIA Live Region for screen readers to provide realtime feedback.   
This behavior has been observed on the graphs on the homepage.

**Comment by sebastianbochan** [comment]
Hi @venkateshpotluri 
Thank you for reporting.

@oysteinmoseng

**Comment by oysteinmoseng** [comment]
Thank you for the feedback, we have more work to do on the accessibility of stock charts. Particularly with charts with lots of data points.

For more information on making Highcharts accessible, including demos of our Accessibility module that enables screen reader features, see https://www.highcharts.com/docs/chart-concepts/accessibility

**Comment by sudheesh001** [comment]
@oysteinmoseng Thanks for the link to the page. It'd be super helpful if the `accessibility.js` ad `export-data.js` can be added to the highcharts homepage too along with the currently available `exporting.js` to show its accessibility capabilities.

I think it's important to mention to developers (in the documentation) who use highcharts that they should consider adding these modules for accessibility purposes. 

_On a side note:_ However, keeping them separate rather than baking it into the product makes it feel like accessibility is an after thought and not mandatory. It'd be so much better to have a web where things are completely accessible irrespective of ability

**Comment by jescrich** [comment]
NVDA or Narrator is not reading the status of toggle visibility button (as disabled or enabled) instead reading as space, when a series is enabled or disabled on a chart.

**Comment by oysteinmoseng** [comment]
@jescrich Tooltips and axes are not announced by design, as we find it would be noisy and not necessary to understand the data for screen reader users. The formatting of these are often tailored to visual users, and could easily be nonsensical to non visual screen reader users. Instead the axes are described in the information region before the chart, and a [customizable label](https://api.highcharts.com/highcharts/plotOptions.series.pointDescriptionFormatter) is read out when users navigate points.

The legend should obviously be included, and this improvement will be rolled out shortly. See #6512 for details.
